packages/manager/modules/emailpro/src/domain/dkim-autoconfig/dkim-auto-configurator.js

```diff
@@ -0,0 +1,133 @@
+export default class DkimAutoConfigurator {
+  initializeScope($scope) {
+    $scope.loadDataForDkim = () => this.loadDataForDkim();
+    $scope.getDkimSelector = () => this.getDkimSelector();
+    $scope.leaveDkimConfigurator = () => this.leaveDkimConfigurator();
+    $scope.getNextButtonDependingOnStep = () =>
+      this.getNextButtonDependingOnStep();
+    $scope.getTitleDependingOnStep = () => this.getTitleDependingOnStep();
+    $scope.initSpfContext = () => this.initSpfContext();
```

Comment:
Is it really necessary to add these functions to the scope ?
Can't you access it through the controller where it's needed ?

Reply:
Yes of course, but I have chosen to keep the same logic as used previously (for configDkim for instance)